.Rbuildignore

```diff
@@ -10,7 +10,7 @@
 ^\.Rproj\.user$
 ^\.idea$
 ^\.dev$
-^\.lintr$
+\.lintr$
```

Comment:
to quiet an `R CMD check` `NOTE` about hidden files -- we have a `.lintr` nested within our test suite

Reply:
Wouldn't this break the tests when running on CRAN source tarballs?

Reply:
Agree it makes sense to have one landing page, but maybe we need

```
+ available linter suites
└-- linters
└-- google-linters
└-- ...
```

instead? ?linters is already 300 lines long

Reply:
Oh, right, since this will mean that doesn't get included. Hmm I guess we just have to live with that note then huh